conjure-go-client/httpclient/client_builder.go

```diff
@@ -130,9 +124,15 @@ func NewClient(params ...ClientParam) (Client, error) {
 
 func getDefaultHTTPClientBuilder() *httpClientBuilder {
 	defaultTLSConfig, _ := tlsconfig.NewClientConfig()
+	uriScorerBuilder := func(uris []string) internal.URIScoringMiddleware {
+		return internal.NewBalancedURIScoringMiddleware(uris, func() int64 {
+			return time.Now().UnixNano()
+		})
+	}
 	return &httpClientBuilder{
 		// These values are primarily pulled from http.DefaultTransport.
 		Proxy:                 http.ProxyFromEnvironment,
+		URIScorerBuilder:      uriScorerBuilder,
```

Comment:
move this outside of the transport settings which as the comment reads were mostly pulled from the stdlib, so I would like to retain the grouping.

Reply:
ping on this one. I noticed you moved it in the struct but not here